### ISSUE TITLE
Feature request: Add file tags (categories) to files in a dataset

### ISSUE THREAD

**Issue by bnavigator** [gdcc/easyDataverse#15]
If I am not mistaken, the `categories` field of a file in a dataset cannot be set by `dataset.add_file()` at the moment and is not part of the `File` class.

https://guides.dataverse.org/en/latest/api/native-api.html#updating-file-metadata

https://github.com/gdcc/easyDataverse/blob/e83579565e65bd28d250cb81c5b3410cd67ec990/easyDataverse/core/dataset.py#L57-L63
https://github.com/gdcc/easyDataverse/blob/e83579565e65bd28d250cb81c5b3410cd67ec990/easyDataverse/core/file.py#L7-L13

**Comment by JR-1991** [comment]
@bnavigator, thanks for submitting the issue! Indeed, that is missing and should definitely be part of the method. I am currently preparing the next version of EasyDataverse and will create a new PR once #16 is merged into the main branch.
